packages/tools/examples/StackSegmentationMammography/index.ts

```diff
@@ -0,0 +1,347 @@
+import { vec3 } from 'gl-matrix';
```

Comment:
can you make this one also use the new API?

Reply:
no I will delete in future versions